alpha/apps/kaltura/lib/v2Tov7Utils.class.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+class v2Tov7Utils
+{
+	const V2TOV7_PARAM_NAME = 'v2tov7';
```

Comment:
I don't really like all these v2tov7. why not have something more intuitive v7CompatibleId or something similar